ios/RNTypeValidation.m

```diff
@@ -0,0 +1,211 @@
+//
+//  RNTypeValidation.m
+//  RNJudo
+//
+//  Copyright (c) 2020 Alternative Payments Ltd
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in all
+//  copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//  SOFTWARE.
+
+#import "RNTypeValidation.h"
+
+@implementation RNTypeValidation
+//----------------------------------------------
+// MARK: - Configs Validation
+//----------------------------------------------
+
++ (nullable NSString *)isConfigurationValid:(NSDictionary *) properties {
+    NSDictionary *configuration = properties[@"configuration"];
+    
+    if(![configuration[@"judoId"] isKindOfClass:[NSString class]]) {
+        return @"judoId property is not valid, judoId should be a string.";
```

Comment:
Type validation is good, it would be nice to check for empty string for Judo ID as well

Reply:
Shouldn't that be done on native sdk ?